tests/system/action/assignment_candidate/test_delete.py

```diff
@@ -95,3 +128,61 @@ def test_delete_finished(self) -> None:
             "It is not permitted to remove a candidate from a finished assignment!",
             response.json["message"],
         )
+
+    def test_delete_no_permission(self) -> None:
+        self.create_meeting()
+        self.user_id = self.create_user("user")
+        self.login(self.user_id)
+        self.set_user_groups(self.user_id, [3])
+        self.set_models(self.permission_test_model)
+        response = self.request("assignment_candidate.delete", {"id": 111})
+        self.assert_status_code(response, 403)
+        assert (
+            "Missing permissions: assignment.can_nominate_other, assignment.can_manage"
+            in response.json["message"]
+        )
+
+    def test_delete_both_permissions(self) -> None:
+        self.create_meeting()
+        self.user_id = self.create_user("user")
+        self.login(self.user_id)
+        self.set_user_groups(self.user_id, [3])
+        self.set_group_permissions(
+            3,
+            [
+                Permissions.Assignment.CAN_NOMINATE_OTHER,
+                Permissions.Assignment.CAN_MANAGE,
+            ],
+        )
+        self.set_models(self.permission_test_model)
+        response = self.request("assignment_candidate.delete", {"id": 111})
+        self.assert_status_code(response, 200)
+
+    def test_delete_both_permissions_self(self) -> None:
+        self.create_meeting()
+        self.user_id = 110
+        self.set_models(self.permission_test_model)
+        self.set_user_groups(self.user_id, [3])
+        self.set_group_permissions(
+            3,
+            [
+                Permissions.Assignment.CAN_NOMINATE_SELF,
+                Permissions.Assignment.CAN_MANAGE,
+            ],
+        )
+        self.login(self.user_id)
+        response = self.request("assignment_candidate.delete", {"id": 111})
+        self.assert_status_code(response, 200)
+
+    def test_delete_no_permissions_self(self) -> None:
+        self.create_meeting()
+        self.user_id = 110
+        self.set_models(self.permission_test_model)
+        self.set_user_groups(self.user_id, [3])
+        self.login(self.user_id)
+        response = self.request("assignment_candidate.delete", {"id": 111})
+        self.assert_status_code(response, 403)
+        assert (
+            "Missing permissions: assignment.can_nominate_self, assignment.can_manage"
+            in response.json["message"]
+        )
```

Comment:
Please also add tests where the phase is not "voting" and the user does not have `CAN_MANAGE` permission.

Reply:
I added a not voting test without Can manage permission.